AutomatedLabDefinition/AutomatedLabDefinition.psm1

```diff
@@ -2498,7 +2498,11 @@ process
     {
         $machine.OperatingSystem = $OperatingSystem
     }
-    
+    elseif ($machine.HostType -eq 'VMWare')
+    {
+        $machine.OperatingSystem = $OperatingSystem
```

Comment:
Ok - will change it to what you proposed. Why i set it up this way, was because i figured it would be convenient to have a separate elsif block for VMWare, so that later additions/logic can be put there.

I have the idea that in a later stage, it may be useful to enumerate the available VM templates and their OSes. Using logic around this:
`(vmware.vimautomation.core\get-vm checex3a).ExtensionData.config | select Name, GuestFullName   # This shows the VMware _configured_ OS (not the _detected_ OS thru VMwareTools, because they are unavailable when machine is poweredoff)